operations/workplace/people/offboarding

```diff
@@ -0,0 +1,39 @@
+**Offboarding Instructions for Managers**
+
+Forward resignation letter/email to MLT member and HR hr@mattermost.com. The resignation letter/email will be stored in Bamboo. 
+A resignation letter/email is required from departing staff. In most cases and depending upon the statutory requirements in the staff member’s region, a minimum of two weeks notice is required for voluntary departures. 
+HR will reach out to the departing staff member to communicate last day instructions and details and to schedule an [exit interview](https://docs.google.com/document/d/1MX6LXa2uVEM3_xqH-pqKyJH4PWis9oJ0a5sAHd58W1U/edit?ts=5e99ecc0).
+
+Confirm Communication Plan with HR: 
+- Staff member’s last day worked (if different than the date in the resignation letter).
+- If needed, provide context on what led to the staff member’s resignation.
+- Confirm rehire eligibility (if resignation was regrettable, not regrettable). 
+- Confirm who will be taking over staff member’s responsibilities in the interim.
+- Confirm whether a backfill is required. HR will reach out to the Recruitment team to kick off discussions about a backfill. 
+- The Team Admin will arrange for the return of company equipment. 
+
+Complete the [Manager Offboarding Checklist](https://docs.google.com/forms/d/1xjLBIuhuy-S6BciNJBTxmWKRbdslPzlrwh7n1T7geQk/edit) to communicate to Security & Biz Ops:
+- Systems that need to be deprovisioned
+- Confirm whether emails need to be forwarded to the member/selected team member for a period following staff member’s departure.
+- Confirm if any company-owned files in Gdrive that need to be transferred to you.
+- Review and approve any outstanding expense reports. 
+
+If applicable, communicate to the Finance Team:
+- Which vendors need to be transitioned (billing approvals, main point of contact). 
+- Confirm status on open procurement requests and transition procurement DRI via procurement@mattermost.com. Include: most recent email exchange and new vendor DRI. 
+- Transfer ownership of any software accounts to Finance or a different team DRI. Notify Systems channel of the change.  
+
+Work with the departing staff member on the messaging around their departure. Confirm timeline of messaging and request that the staff member refrain from sharing info ahead of the planned timeline
+Communicate the agreed upon messaging around the departure with your team, department, and any other relevant groups as soon as possible. This is important to reinforce asynchronous communication. Silence and/or lack of communication can lead to confusion, misalignment, conjecture, gossip, and insecurity. Include the departing staff member in the announcement if they are comfortable participating in the discussion.
+To avoid “@ all” departure emails/Mattermost messages, invite the departing staff member to draft a message to be shared in the [Stand-Up Channel](https://community.mattermost.com/private-core/channels/stand-up). This message should be reviewed by you and HR. 
```

Comment:
```suggestion
To avoid “@ all” departure emails/Mattermost messages, invite the departing staff member to draft a message to be shared in the [Stand-Up Channel](https://community.mattermost.com/private-core/channels/stand-up). This message should be reviewed by the staff member's manager and HR. 
```